app/javascript/react/screens/App/common/ConfirmModal.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
```

Comment:
nice one! look forward to seeing this in PF-R!

Reply:
Will do! I have that PR almost ready, but was having some dumb issues with it. I'll push tomorrow.